shopcart-backend/src/main/java/com/shopcart/backend/service/DummyPaymentService.java

```diff
@@ -0,0 +1,23 @@
+package com.shopcart.backend.service;
+
+public class DummyPaymentService {
+
+    private static DummyPaymentService instance = null;
+	public static int count = 0;
+
+    public static DummyPaymentService getInstance() {
```

Comment:
Good singleton but run a format on this file to clean it up a little

Reply:
Applied